Monika After Story/game/script-topics.rpy

```diff
@@ -8404,6 +8404,87 @@ label monika_whydoyouloveme:
 #    m 1k "That's why I love you, [player]!"
 #    return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_bored",
+            category=['misc'],
+            random=True,
+            prompt="Boredom"
```

Comment:
Thinking now that it's not written as bored anymore, this needs a different prompt.  Something like "Let's play a game"